src/test/scala/org/broadinstitute/dsde/firecloud/service/NihApiServiceSpec.scala

```diff
@@ -35,63 +46,85 @@ class NihApiServiceSpec extends ApiServiceSpec {
   }
 
   it should "return BadRequest when NIH linking with an invalid JWT" in withDefaultApiServices { services =>
-    Post("/nih/callback", JWTWrapper("bad-token")) ~> dummyUserIdHeaders(services.thurloeDao.normalUserId) ~> sealRoute(services.nihRoutes) ~> check {
+    Post("/nih/callback", JWTWrapper("bad-token")) ~> dummyUserIdHeaders(services.thurloeDao.TCGA_AND_TARGET_UNLINKED) ~> sealRoute(services.nihRoutes) ~> check {
       status should equal(BadRequest)
     }
-    assert(!services.rawlsDao.groups(dbGapAuthorizedUsersGroupName).contains(services.thurloeDao.normalUserId))
+    assert(!services.rawlsDao.groups(tcgaDbGaPAuthorized).contains(services.thurloeDao.TCGA_AND_TARGET_UNLINKED))
+    assert(!services.rawlsDao.groups(targetDbGaPAuthorized).contains(services.thurloeDao.TCGA_AND_TARGET_UNLINKED))
+  }
+
+  it should "link and sync when user is on TCGA whitelist but not TARGET" in withDefaultApiServices { services =>
+    assert(!services.rawlsDao.groups(targetDbGaPAuthorized).contains(services.thurloeDao.TCGA_UNLINKED))
```

Comment:
Please add something like `val toLink = services.thurloeDao.TCGA_UNLINKED` to make the assertions read more naturally.